### ISSUE TITLE
Cannot get the schema - validation issue

### ISSUE THREAD

**Issue by pawelprazak** [lbrlabs/pulumi-lbrlabs-eks#110]
I was not able to get the schema using pulumi CLI:
```
pulumi --non-interactive --logtostderr package get-schema lbrlabs-eks@0.16.0
error: <nil>: #/types/lbrlabs-eks:index:IngressConfig: doesn't validate with '/$defs/complexTypeSpec'; , and 19 other diagnostic(s)
```

**Comment by jaxxstorm** [comment]
I'm unsure of how to resolve this, it works generally for other packages?

**Comment by jaxxstorm** [comment]
This has been resolved as of `v0.20.0`
